lib/homekit-device.js

```diff
@@ -2846,7 +2825,7 @@ class HomeKitDevice extends Device {
         this.paired = true;
         this.pinRequired = false;
         database.close();
-        return this.addDevicesAndProperties();
+        return this.addDevicesAndProperties(true);
```

Comment:
nit: I've always hated boolean parameters because you can't tell what true means from the call site.

I think that the code would read better by creating a SKIP_QUEUE constant equated to true and then doing:
```
return this.addDevicesAndProperties(SKIP_QUEUE);
```

Similar comments for the other functions

Reply:
Done.